### DISCUSSION TITLE
Cannot add new task (even if it is less than limit)

### DISCUSSION THREAD

**Discussion by tayyebi** [kanboard/kanboard#5214] [Installation & Configuration]
![image](https://user-images.githubusercontent.com/14053493/168461512-84685e10-3fb8-4fb6-97da-ccac9ff75ac9.png)

When I click 'add new', it will not add a new task. But other features are working. I have to remove a task in board (or other boards) to let me add a new task. I didn't find any option in settings for open tasks limitation. Is it a bug? And after several times clicking on the button, it will block my IP for some minutes.

I also tried removing cookies and persistent connections.

Thanks.

**Comment by tayyebi** [comment]
This is my apache error log
```
2022-05-15 21:36:23.694297 [INFO] [50207] [T0] [5.127.23.59:37662:HTTP2-55#APVH_team.gordarg.com:443] File not found [/home2/notlocal/public_html/team/403.shtml] 
```

**Comment by tayyebi** [comment]
I also wrote a script to change permissions of files to ensure that it's not a server problem.

```
<?php

if ( ! function_exists('glob_recursive'))
{
    // Does not support flag GLOB_BRACE        
   function glob_recursive($pattern, $flags = 0)
   {
     $files = glob($pattern, $flags);
     foreach (glob(dirname($pattern).'/*', GLOB_ONLYDIR|GLOB_NOSORT) as $dir)
     {
       $files = array_merge($files, glob_recursive($dir.'/'.basename($pattern), $flags));
     }
     return $files;
   }
}

$path = '*.php';

$filenames = glob_recursive($path);

foreach ($filenames as $filename) {
    echo  $filename . '<br>';
    chmod($filename,0755);
}

?>
```

rights: <https://stackoverflow.com/questions/12109042/>

**Comment by RainerBielefeld** [comment]
@[tayyebi](https://github.com/tayyebi):
Still a problem for you?

**Comment by tayyebi** [comment]
It was a million years ago. I can't remember. But I'm using it in container these days with SQLite and it's working like a charm. Thank you and well done. <3
